packages/ringcentral-integration/modules/ConferenceCall/index.js

```diff
@@ -784,6 +788,36 @@ export default class ConferenceCall extends RcModule {
     };
   }
 
```

Comment:
this onMerge function is basically copied from CallCtrlPage container to support proxy mechanism.